src/styles/main.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+$cell-width: 100px;
+$cell-height: 100px;
+$border-size: 1px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 750px;
+  position: absolute;
+  
+  @for $index from 1 through 31 {
+    :nth-child(#{$index})::before {
+      display: flex;
+      justify-content: center;
+      height: 100px;
+      line-height: 100px;
```

Comment:
You can move `display: flex; justify-content: center;` and apply `align-items: center` to `calendar__day `class and remove all this classes from here